scripts/chief_deploy.py

```diff
@@ -109,19 +109,19 @@ def setup_dependencies(ctx):
         # Creating a virtualenv tries to open virtualenv/bin/python for
         # writing, but because virtualenv is using it, it fails.
         # So we delete it and let virtualenv create a new one.
-        python = os.path.join(VENV_BIN, 'python')
-        python27 = os.path.join(VENV_BIN, 'python2.7')
-        ctx.local('rm -f %s' % python)
-        ctx.local('rm -f %s' % python27)
+        ctx.local('rm -rf %s' % settings.VENV_DIR)
         ctx.local('virtualenv-2.7 --no-site-packages %s' % settings.VENV_DIR)
 
         # Activate virtualenv to append to the correct path to $PATH.
         activate_env = os.path.join(VENV_BIN, 'activate_this.py')
         execfile(activate_env, dict(__file__=activate_env))
 
+        ctx.local('%s install --upgrade pip' % os.path.join(VENV_BIN, 'pip'))
         ctx.local('pip --version')
-        ctx.local('%s scripts/peep.py install -r requirements/default.txt' %
-                  python)
+        python = os.path.join(VENV_BIN, 'python')
+        ctx.local('%s scripts/peep.py install '
+                  '-r requirements/default.txt'
```

Comment:
You're concatenating strnigs here, so I think you need a space after `default.txt`.